src/app/components/MediaLoader/index.tsx

```diff
@@ -235,40 +245,54 @@ const MediaLoader = ({ blocks, className, embedded }: Props) => {
           <Metadata blocks={blocks} embedURL={playerConfig?.externalEmbedUrl} />
         )
       }
-      {orientation === 'portrait' && (
+      {showPortraitTitle && (
         <strong css={styles.titlePortrait}>Watch Moments</strong>
       )}
       <figure
         data-e2e="media-loader__container"
-        css={styles.figure}
         className={className}
+        css={[
+          styles.figure(embedded),
+          setAspectRatio && [
+            orientation === 'portrait' && styles.portraitFigure(embedded),
```

Comment:
We need to pass in `embedded` into the stylesheets because AMP will iframe and ultimately render this component. So some of the styles should _only_ apply when to the outer `figure` and not the inner `figure` inside the iframe.